flask/app/datasets.py

```diff
@@ -177,26 +197,32 @@ def update_dataset(id: int):
             .first_or_404()
         )
     else:
+        app.logger.debug("Processing query - unrestricted based on user id")
         dataset = models.Dataset.query.filter_by(dataset_id=id).first_or_404()
 
     enum_error = mixin(dataset, request.json, editable_columns)
 
     if enum_error:
+        app.logger.error("%s", enum_error)
         abort(400, description=enum_error)
```

Comment:
Looks good to me, not a big deal but I think we could add the following to the enum checks so it's in line with the rest of the endpoints
```suggestion
    app.logger.debug("Validating enums..")
    enum_error = mixin(dataset, request.json, editable_columns)

    if enum_error:
        app.logger.error("%s", enum_error)
        abort(400, description=enum_error)
    else:
        app.logger.debug("All enums supplied are valid.")
```